examples/node-app-mssql-ts/Dockerfile

```
@@ -1,9 +1,9 @@
-FROM node:10
+FROM node:12
```

For the sake of consistency.

```suggestion
FROM node:14-slim
```